api/serverpb/server.proto

```diff
@@ -181,7 +182,18 @@ message GetSettingsResponse {
 }
 
 message ChangeSettingsRequest {
-  bool enable_telemetry = 1;
+  option (grpc.gateway.protoc_gen_swagger.options.openapiv2_schema) = {
+    json_schema: {
+      title: "Super puper message title!"
+      description: "Awesome message description"
+    }
+  };
```

Comment:
Message title and descriptions

Reply:
But do we need them? Messages don't really exist in JSON APIs

Reply:
That's more like a comment to the request body (see screenshot), and I suppose we don't need them. I just tried what we can to specify.

Most likely we need only method descriptions.